plugin/mlsdbprovider.cpp

```diff
@@ -322,6 +324,11 @@ void MlsdbProvider::tryFetchOnlinePosition()
     updateLocationFromCells(cellIds);
 }
 
+void MlsdbProvider::onlineWifiChanged()
+{
+    tryFetchOnlinePosition();
```

Comment:
I'm a bit afraid how often the signal can trigger on a place with lots and lots of networks. There's also m_recalculatePositionTimer so I'm not sure how much benefit there would be over sending even more requests.

Ping @chriadam too

Reply:
Phone p4903 gives information to one cellular tower on one SIM card. In this case, the service can not give the correct data. For example, on the phone p4903, the signal onlineWifiChanged signal does not trigger very often, and the signal cellularNetworkRegistrationChanged is very rare. In the phone p4903 without the signal onlineWifiChanged service is almost not needed.